nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow_schema.json

```diff
@@ -5,195 +5,237 @@
-            "description": "Custom config file to supply to MultiQC",
-            "default": "",
-            "fa_icon": "fas fa-cog",
-            "hidden": true
-        },
-        "max_multiqc_email_size": {
-            "type": "string",
-            "description": "File size limit when attaching MultiQC reports to summary emails",
-            "default": "25 MB",
-            "fa_icon": "fas fa-file-upload",
-            "hidden": true
-        },
-        "publish_dir_mode": {
-            "type": "string",
-            "default": "copy",
-            "hidden": true,
-            "description": "Method used to save pipeline results to output directory",
-            "help_text": "The Nextflow `publishDir` option specifies which intermediate files should be saved to the output directory. This option tells the pipeline what method should be used to move these files. See https://www.nextflow.io/docs/latest/process.html#publishdir for details.",
-            "fa_icon": "fas fa-copy",
-            "enum": [
-                "symlink",
-                "rellink",
-                "link",
-                "copy",
-                "copyNoFollow",
-                "mov"
-            ]
-        },
-        "monochrome_logs": {
-            "type": "boolean",
-            "description": "Do not use coloured log outputs",
-            "fa_icon": "fas fa-palette",
-            "hidden": true,
-            "default": false
+        "Generic options": {
+            "type": "object",
+            "properties": {
+                "reads": {
+                    "type": "string",
+                    "default": "data/*{1,2}.fastq.gz",
+                    "fa_icon": "fas fa-dna",
+                    "description": "Input FastQ files.",
+                    "help_text": "A glob pattern for input FastQ files. Should include at least one asterisk (*). For paired-end data, should contain curly brackets with two patterns differentiating the paired reads e.g. `*_R{1,2}.fastq.gz`"
+                },
+                "single_end": {
+                    "type": "boolean",
+                    "description": "Specifies that the input is single-end reads.",
+                    "fa_icon": "fas fa-align-center",
+                    "default": false,
+                    "help_text": "By default, the pipeline expects paired-end data. If you have single-end data, specify this parameter on the command line when you launch the pipeline. It is not possible to run a mixture of single-end and paired-end files in one run."
+                }
+            },
+            "required": [
+                "reads"
+            ],
+            "fa_icon": "fas fa-terminal"
         },
-        "tracedir": {
-            "type": "string",
-            "description": "Directory to keep pipeline Nextflow logs and reports",
-            "default": "./results/pipeline_info",
-            "fa_icon": "fas fa-cogs",
-            "hidden": true
+        "Reference genome options": {
```

Comment:
This group only has one non-hidden parameter in it. So I don't think it should really be a group..

Reply:
Yeah, but if there will certainly be other reference genome parameters that get added so its quite nice to have a separate group for this:
https://github.com/drpatelh/nf-core-atacseq/blob/999c8b3925de895a9951422730d18d677a29e4ac/nextflow_schema.json#L49-L128

Reply:
ok 👌